server/config.py

```diff
@@ -57,42 +76,30 @@ def refresh(self):
 
         entity = config[0]
         if KEY_TRAINING_ENABLED in entity:
-            self.training_enabled = entity[KEY_TRAINING_ENABLED]
+            self[KEY_TRAINING_ENABLED] = entity[KEY_TRAINING_ENABLED]
```

Comment:
Can you add a method like this?
```
def __set_value(self, entity, key, default_value):
    self[key] = entity[key] if key in entity else default_value
```
And then call it here. Or just replace each of these if/else with
```
self[key] = entity[key] if key in entity else <default value>
```